airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/DefaultJobCreator.java

```diff
@@ -24,10 +25,14 @@
 
   private final JobPersistence jobPersistence;
   private final ConfigRepository configRepository;
+  private final ResourceRequirements workerResourceRequirements;
 
-  public DefaultJobCreator(final JobPersistence jobPersistence, final ConfigRepository configRepository) {
+  public DefaultJobCreator(final JobPersistence jobPersistence,
```

Comment:
This is the class with the main changes of using the WorkerConfig resource requirements if they are not set on the StandardSync